WalletWasabi.Fluent/Behaviors/DragWindowBehavior.cs

```diff
@@ -0,0 +1,35 @@
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.VisualTree;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class DragWindowBehavior : AttachedToVisualTreeBehavior<Control>
+{
+	private IDisposable? _onDrag;
+
+	protected override void OnAttachedToVisualTree(CompositeDisposable disposable)
+	{
+		if (AssociatedObject is null)
+		{
+			return;
+		}
+
+		if (AssociatedObject.FindAncestorOfType<Window>() is not { } window)
+		{
+			return;
+		}
+
+		_onDrag =
+			Observable.FromEventPattern<PointerPressedEventArgs>(AssociatedObject, nameof(Control.PointerPressed))
+					  .Where(e => e.EventArgs.InputModifiers == InputModifiers.LeftMouseButton)
```

Comment:
Can you fix the `obsolete` warning here?

Reply:
@soosr Done ;)

I didn't look hard enough to find the replacement. I found a `PointerPointProperties` property in there but it is `protected`, so I couldn't use it. Now I found the correct one